src/cmd/cli/command/commands.go

```diff
@@ -478,9 +478,20 @@ var generateCmd = &cobra.Command{
 				}
 				if sample == generateWithAI {
 					sample = ""
-					defaultFolder = "project1"
-				} else {
-					defaultFolder = sample
```

Comment:
What's the current behaviour if the directory exists? Do we just fail with the OS error?

Reply:
Right now, we ignore the error from `os.Mkdir`, but issue warnings when files are overwritten.